bitnami/metallb/templates/secret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+data:
+  secretkey: {{ randAlphaNum 256 | b64enc | quote }}
+kind: Secret
+metadata:
+  name: {{ include "metallb.fullname" . }}-memberlist
+  labels: {{- include "metallb.labels" . | nindent 4 }}
+    component: speaker
+  annotations:
+    "helm.sh/hook": "pre-install"
+    "helm.sh/hook-delete-policy": "before-hook-creation"
```

Comment:
Not sure I catch your point yet, I can see on that link the secret is created after deploying the main manifest

Reply:
Not sure I catch your point yet, I can see on that link the secret is created after deploying the main manifest

Reply:
correct. I automated only that step. So that a helm install is working without doing anything manually regarding secrets.
It also ensures that the secret is not updated on a helm upgrade.